### PR TITLE
add DefaultActions system

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
title
adds an action that is expected to always exist

should be used for specific entities and not things like scream or stating laws which would be for a component shared across multiple entities

## Why / Balance
- exterminator freedom action is why i made this
- maybe revenant/ling/xeno for the store action, or could be done better with a dedicated intrinsic store component
- probably aghost with all the consoles would be way nicer to have them in yml instead of demonic stuff, would also make adding manifest / whatever actions easy

## Technical details
basically the boilerplate but defined in yml instead of adding yet another XAction XActionEntity mapinit combo to your specific entity's specific component

ActionPair might be reusable for the rest of action api but thats for the future

usage is simple
```yml
- type: DefaultActions
  actions:
  - id: BoringAction
  - id: CoolAction
```

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
no cl no fun

### CODE REVIEW

**Comment by deltanedas** [comment]
if you have an event that is handled by a generic system (like a breaking free event that can work for terminator or ling, different message in event) you shouldnt need to have a changeling component on terminator or vice versa to do that, or have a random action only used by 2 things in Cuffable

**Comment by deltanedas** [comment]
that can already be done but it means having a lot of duplication with the same code over and over, and means more hardcoding for more actions instead of adding a single line in yml
